protected/humhub/libs/BaseSetting.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * @link      https://www.humhub.org/
+ * @copyright Copyright (c) 2017 HumHub GmbH & Co. KG
+ * @license   https://www.humhub.com/licences
+ */
+
+namespace humhub\libs;
+
+use Yii;
+use yii\db\ActiveRecord;
+
+/**
+ * BaseSetting
+ *
+ * @since  1.13.2
+ * @author Martin Rüegg <martin.rueegg@metaworx.ch>
+ */
+abstract class BaseSetting
```

Comment:
```suggestion
abstract class SettingActiveRecord
```

For base ActiveRecord classes, we currently always add the suffix "ActiveRecord". Normally we also add these in the `components` namespace, but in this case `libs` is also fine.